### ISSUE TITLE
'Working with objects' refresher for API week

### ISSUE THREAD

**Issue by jackherizsmith** [foundersandcoders/coursebook#229]
People felt rusty on objects for API week, perhaps it could at least be introduced that different APIs respond with all shapes and sizes of objects - perhaps a workshop where the requests and json etc is all set up, and they're tasked to find and console.log a particular thing hidden within a response?

**Comment by oliverjam** [comment]
Hmm, what exactly do you think would be useful here? Accessing object properties with dot/square bracket notation. I guess a more advanced thing would be looping over properties, but that shouldn't be something they need for API responses.

The only slot that isn't really _necessary_ in this week's schedule is the DOM manipulation morning challenge, but I think it's important to cover DOM stuff (since that's not really in the pre-reqs at all).

**Comment by jackherizsmith** [comment]
I just mean basic JS object navigation, @Albadylic made the good point that this is already covered in the pre-reqs. Perhaps as part of the intro to the project, a mentor could spend two minutes reading docs then writing a very quick fetch (just in their browser console), e.g. to pokeapi, and get an object back to then explore what's inside?

i.e.

![image](https://user-images.githubusercontent.com/58533031/89189312-b6179c00-d597-11ea-952c-063a083b368f.png)

Basically a team tried to use Wikimedia API and it's about as confusing as it gets.

**Comment by oliverjam** [comment]
Yeahhh that shouldn't be on the recommended list, it's so complex.

I think to a certain extent we have to accept that the projects are where the majority of the learning happens—it's _good_ if they're struggling with stuff and figuring it out as they go. We can't keep going back over the same stuff every week to make sure no one will have any problems at all (especially things from the pre-reqs).

If the current fetch workshops don't have an example where they have to access data from the response then we should definitely add that though

**Comment by jackherizsmith** [comment]
I think this is part three of the workshop [here](https://github.com/oliverjam/real-world-fetch/blob/master/README.md). Will close this.
